frontend/src/Pages/AllUser/Login/ResetPassword.jsx

```
@@ -0,0 +1,167 @@
+
+    return (
+        <div className="flex items-center justify-center min-h-screen bg-gray-900 text-white px-6">
+            <div className="bg-gray-800 shadow-lg rounded-lg p-8 w-full max-w-md">
+                <h2 className="text-2xl font-semibold text-center text-gray-200 mb-4">Reset Your Password</h2>
+
+                {message && <p className="text-green-500 text-center">{message}</p>}
+                {error && <p className="text-red-500 text-center">{error}</p>}
+
+                <form onSubmit={handleSubmit} className="space-y-6">
+                    <div>
+                        <label className="block text-sm font-medium text-gray-300">New Password</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)